src/de/jost_net/JVerein/gui/action/RechnungAction.java

```diff
@@ -0,0 +1,30 @@
+/**********************************************************************
+ * Copyright (c) by Heiner Jostkleigrewe
+ * This program is free software: you can redistribute it and/or modify it under the terms of the 
+ * GNU General Public License as published by the Free Software Foundation, either version 3 of the 
+ * License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,  but WITHOUT ANY WARRANTY; without 
+ *  even the implied warranty of  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See 
+ *  the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program.  If not, 
+ * see <http://www.gnu.org/licenses/>.
+ * 
+ * heiner@jverein.de
+ * www.jverein.de
+ **********************************************************************/
+package de.jost_net.JVerein.gui.action;
+
+import de.jost_net.JVerein.gui.view.RechnungView;
+import de.willuhn.jameica.gui.Action;
+import de.willuhn.jameica.gui.GUI;
+
+public class RechnungAction implements Action
+{
+  @Override
+  public void handleAction(Object context)
+  {
+    GUI.startView(RechnungView.class.getName(), context);
```

Comment:
Davor sollte geprüft werden ob context eine Rechnung ist.

Reply:
In welchem Fall sollte es denn keine Rechnung sein? Es kann ja nur über das Kontextmenu aufgerufen werden.

Reply:
Das habe ich in meinem Kommentar beschrieben. Ich hatte einen View ohne Einträge. Wenn man den View öffnet ist das Menü bei rechter Maus aktiv. Es ist aber nichts selektiert. Bein klick auf den Menüpunkt bekam ich dann die Exception.
Wenn man einmal links klickt bleibt das Menü auch bei rechter Maustaste disabled.
Ich hatte schon ähnliche Probleme und darum mag ich eigentlich auch nicht gerne wenn bei manchen Edit Actions wenn nichts selektiert wurde einfach eine neue Instanz erzeugt wird. Das wollte man ja eigentlich nicht. Dafür gibt's ja meist die Neu Action.

Reply:
erledigt